libtenzir/builtins/operators/http.cpp

```diff
@@ -407,17 +426,778 @@ struct from_http final : public virtual operator_plugin2<from_http_operator> {
 
   auto make(invocation inv, session ctx) const
     -> failure_or<operator_ptr> override {
-    auto args = http_args{};
+    auto args = from_http_args{};
     args.op = inv.self.get_location();
     auto p = argument_parser2::operator_(name());
     args.add_to(p);
     TRY(p.parse(inv, ctx));
     TRY(args.validate(ctx));
     return std::make_unique<from_http_operator>(std::move(args));
   }
+
+  auto load_properties() const -> load_properties_t override {
+    return {
+      .schemes = {"http", "https"},
+      // .default_format = plugins::find<operator_factory_plugin>("read_json"),
```

Comment:
Switching `from "http://…"` from `load_http` to `from_http` should be accompanied by deprecating `load_http` and a changelog entry.

Reply:
Oops, this will be a following PR.